src/PatientChecking/PatientChecking/Controllers/AppointmentController.cs

```diff
@@ -1,16 +1,57 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using PatientChecking.Services.Repository;
+using PatientChecking.Services.ServiceModels;
+using PatientChecking.Services.ServiceModels.Enum;
+using PatientChecking.Views.ViewModels;
 using System;
 using System.Collections.Generic;
+using System.Dynamic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace PatientChecking.Controllers
 {
     public class AppointmentController : BaseController
     {
-        public IActionResult Index()
+        private readonly IAppointmentService _appointmentService;
+
+        public AppointmentController(IAppointmentService appointmentService)
+        {
+            _appointmentService = appointmentService;
+        }
+
+        public async Task<IActionResult> Index(int Option = (int)AppointmentSortSelection.ID, int PageSize = 10, int PageIndex = 1)
         {
-            return View();
+            PagingRequest request = new PagingRequest()
+            {
+                PageIndex = PageIndex,
+                PageSize = PageSize,
+                SortSelection = Option
+            };
+            AppointmentList pagedResult = await _appointmentService.GetListAppoinmentsPaging(request);
+            List<AppointmentViewModel> appointmentViewModels = new List<AppointmentViewModel>();
+            foreach(Appointment appointment in pagedResult.Appointments)
```

Comment:
Should add empty line above for readability. Please check for other places.

Reply:
Added empty line above for readability